account_bank_statement_import_camt_details/models/account_bank_statement_line.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+"""Add process_camt method to account.bank.statement.import."""
+# © 2017 Compassion CH <http://www.compassion.ch>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, fields
+
+
+class AccountBankStatementLine(models.Model):
+    """Add process_camt method to account.bank.statement.import."""
+    _inherit = 'account.bank.statement.line'
+
+    partner_address = fields.Char()
+    partner_account = fields.Char()
+    partner_bic = fields.Char()
```

Comment:
The IBAN is already encoded in `bank_account_id` thanks to [https://github.com/OCA/bank-statement-import/blob/9.0/account_bank_statement_import_camt/models/parser.py#L98](url)

This is in addition so that if the bank account is not matched to a known partner, we can still see the account information in the reconcile view. By default they are not showing in view.